### ISSUE TITLE
Use config files for kube-proxy instead of command line args

### ISSUE THREAD

**Issue by surajssd** [kinvolk/lokomotive#1316]
Currently kube-proxy throws such warnings in its logs:

```
W0113 12:08:48.344275       1 server.go:226] WARNING: all flags other than --config, --write-config-to, and
--cleanup are deprecated. Please begin using a config file ASAP.
```

Fix the above warning by using the config files mounted from configmap.

**Comment by knrt10** [comment]
Related https://github.com/kinvolk/lokomotive/issues/1284

**Comment by invidian** [comment]
Note that the configuration file is still in alpha: https://pkg.go.dev/k8s.io/kube-proxy@v0.20.2/config/v1alpha1. Do we want to rely on alpha features?

**Comment by surajssd** [comment]
> Related #1284

@knrt10 isn't it kubelet only? While this issue deals with kube-proxy!

> Note that the configuration file is still in alpha: https://pkg.go.dev/k8s.io/kube-proxy@v0.20.2/config/v1alpha1. Do we want to rely on alpha features?

That **ASAP** in the comment really got me.

**Comment by knrt10** [comment]
> @knrt10 isn't it kubelet only? While this issue deals with kube-proxy!

Yes, I meant related not duplicate :-). Someone taking up any of the issue can close both at the same time
